client/my-sites/plan-features/item.jsx

```diff
@@ -27,15 +28,25 @@ export default function PlanFeaturesItem( {
 		onMouseLeave( event.currentTarget, description );
 	};
 
+	const mouseEvents = {
+		onMouseEnter: handleOnMouseEvent,
+		onMouseLeave: handleOnMouseLeave
+	};
+	const hoverOnRow = (
+		abtest( 'plansDescriptions' ) === 'ascendingPriceEagerDescription' ||
+		abtest( 'plansDescriptions' ) === 'descendingPriceEagerDescription'
+	);
+
 	return (
-		<div className="plan-features__item">
+		<div className="plan-features__item"
+			{ ...( hoverOnRow && mouseEvents ) }
```

Comment:
Keep in mind that the target element (the `context` Popover property) is gotten from the function bound to the event.
In this case the function is `handleOnMouseEvent`, the event is `onMouseEnter` and the target is `<div className="plan-features__item">`.
For this reason the Popover isn't anchored to the `(i)` element.

![image](https://cloud.githubusercontent.com/assets/77539/17928782/d1edc290-69d2-11e6-85c9-124d3ec8eb54.png)